tests/routes/test_search.py

```diff
@@ -1264,7 +1265,8 @@ def test_csv_content(
                     )
                 else:
                     assert (
-                        document["document_source_url"] == row["Document Content URL"]
+                        document["document_source_url"]
```

Comment:
Is there a way to make this more explicit about what we are testing for. The code is fine but I'm not sure what we are asserting for - the complexity of the statement here is a little obtuse for me - or maybe we should add a comment?

Reply:
Perfect :ok: thanks